Simplified/Reader2/UI/NYPLReaderSettingsView.swift

```diff
@@ -0,0 +1,723 @@
+    button.accessibilityLabel = NSLocalizedString("SerifFont", comment: "Accessible label for the font")
+    button.addTarget(self, action: #selector(didSelectSerif), for: .touchUpInside)
+    return button
+  }()
+  
+  private lazy var openDyslexicButton: UIButton = {
+    let title = NSLocalizedString("AlphabetFontType", comment: "A title for presenting the desired font")
+    
+    let button = newCustomButton(title: title,
+                                 titleColor: .black,
+                                 fontName: "OpenDyslexic3",
+                                 fontSize: 20,
+                                 normalStateAttributes: [NSAttributedString.Key.underlineStyle: 0],
+                                 disabledStateAttributes: [NSAttributedString.Key.underlineStyle: 1])
+    button.contentVerticalAlignment = .top
+    button.accessibilityLabel = NSLocalizedString("OpenDyslexicFont", comment: "Accessible label for the font")
+    button.addTarget(self, action: #selector(didSelectOpenDyslexic), for: .touchUpInside)
+    return button
+  }()
+  
+  private lazy var publisherDefaultButton: UIButton = {
+    let title = NSLocalizedString("PublisherDefaultButtonTitle", comment: "A title for publisher's default font selection")
+    let button = newCustomButton(title: title,
+                                 titleColor: .black,
+                                 fontName: "Helvetica",
+                                 fontSize: 24,
+                                 normalStateAttributes: [NSAttributedString.Key.underlineStyle: 0],
+                                 disabledStateAttributes: [NSAttributedString.Key.underlineStyle: 1])
+    button.accessibilityLabel = NSLocalizedString("Publisher's Default Font", comment: "Accessible label for the font")
+    button.addTarget(self, action: #selector(didSelectPublisherDefault), for: .touchUpInside)
+    return button
+  }()
+  
+  private lazy var selectedFontBottomBorder: UIView = {
+    let view = UIView()
+    view.backgroundColor = NYPLConfiguration.backgroundColor()
+    return view
+  }()
+  
+  private lazy var fontTitleLabel: UILabel = {
+    let label = UILabel()
+    label.textAlignment = .left
+    label.font = .systemFont(ofSize: 22, weight: .medium)
+    label.text = NSLocalizedString("SpecialFontDescriptionTitle", comment: "Font description title")
+    label.numberOfLines = 1
+    return label
+  }()
+  
+  private lazy var fontDescriptionLabel: UILabel = {
+    let label = UILabel()
+    label.textAlignment = .left
+    label.font = .systemFont(ofSize: 16)
+    label.numberOfLines = 0
+    // NYPLReaderSettingsView is dynamic height when being initialized,
+    // so we want to initialize this label with the possible largest amount of text
+    // to ensure there is enough room.
+    label.text = NSLocalizedString("PublisherDefaultDescriptionContent", comment: "Publisher's default description")
+    label.autoSetDimension(.height, toSize: 50)
```

Comment:
will this size scale if the label text has, for example, 5 lines or more, depending on the localization? from here it seems like `50` is a fixed size, which could be a problem.